src/documentation/common/ShowMoreLink.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * (c) 2021, Micro:bit Educational Foundation and contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+import { Link, LinkProps } from "@chakra-ui/react";
+import { FormattedMessage } from "react-intl";
+
+interface ShowMoreLinkProps extends LinkProps {
+  isOpen: boolean;
+}
+
+const ShowMoreLink = ({ isOpen, ...props }: ShowMoreLinkProps) => (
+  <Link {...props} color="brand.600">
```

Comment:
I think it should still actually be a button (maybe via `as="button"`).

Reply:
I've also changed the name back to `ShowMoreButton` to match - [`ef4d9e9`](https://github.com/microbit-foundation/python-editor-next/pull/704/commits/ef4d9e90f093b05b01029fefe1a76bc2d874c716)